Makefile

```diff
@@ -1,3 +1,5 @@
+.PHONY: version up-schema down-schema force-schema lint lint-verbose
```

Comment:
Personally I find is easier to just set the PHONY for each target, to make sure it's not forgotten about/unmaintained.

Example:

```Makefile
test:
	@go test -cover -race ./...
.PHONY: test

lint:
	@golangci-lint run --config=.golangci.yml ./...
.PHONY: lint
```

Reply:
Alright ! Love this way too, it's more readable 👍